src/WalkingModule/src/Module.cpp

```diff
@@ -28,8 +28,15 @@
 #include <WalkingControllers/YarpUtilities/Helper.h>
 #include <WalkingControllers/StdUtilities/Helper.h>
 
+
 using namespace WalkingControllers;
 
+//timeOffset is the time of start of this step(that will be updated in updateTrajectory function at starting point of each step)
+double timeOffset;
+double impactTimeNominal, impactTimeAdjusted;
+iDynTree::Vector2 zmpNominal, zmpAdjusted;
+int indexPush;
```

Comment:
Please move these parameters within the class. Using global variables can generate a lot of problems.

Reply:
Done!